mungegithub/mungers/publish_scripts/publish_api.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script publishes the latest changes in the ${src_branch} of
+# k8s.io/kubernetes/staging/src/k8s.io/api to the ${dst_branch} of
+# k8s.io/kube-aggregator.
```

Comment:
The first sed was global :) Can you scrape the whole comment you have here and replace all approprirate instances?